### PR TITLE
Generated CLI doc update for release 0.2.5 (Travis build: 232)

### PR DESCRIPTION
**Pull request by kylegc**

### CODE REVIEW

File: content/docs/using-appsody/cli-commands.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
+title: CLI Reference
 path: /docs/using-appsody/cli-commands
+section: Using Appsody
```
Comment (tomleah) [inline content/docs/using-appsody/cli-commands.md]: `title` and `section` are no longer needed in front matter

Comment (kylegc) [reply content/docs/using-appsody/cli-commands.md]: Hi @tomleah, will having these break the website?
If not, I suggest we merge this now and create a new issue to update the CLI doc generator for next release.

Comment (tomleah) [reply content/docs/using-appsody/cli-commands.md]: Agreed, will merge this now 🙂

Comment (kylegc) [reply content/docs/using-appsody/cli-commands.md]: Thanks. I created issue https://github.com/appsody/appsody/issues/87

**Pull request merged**